BSP/Simulator/Hardware/UART.py

```diff
@@ -0,0 +1,42 @@
+import csv
+import os
+import fcntl
+
+# path to the file
+file = "BSP/Simulator/Hardware/Data/UART.csv"
+
+
+def write(msg, uart_num):
```

Comment:
This module would only be used for UART_2 right now since UART_1 is the display, so it can just assume 2 and not take a parameter.